app/routes.py

```diff
@@ -179,3 +181,14 @@ def prepare_validation_response(request, data):
         'data': data,
     }
     return ValidationResponse(**response_data, media_type="application/json")
+
+
+@router.get("/ags_log/{bgs_loca_id}")
```

Comment:
This decorator needs to specify the `response_class` and `responses` for the openapi. You'll need top create a `pdf_responses` dictionary - see the other routes and the top of the file for examples.

You should also either use or create an error class (see `errors.py`) for a full error response.

Reply:
Started in https://github.com/BritishGeologicalSurvey/pyagsapi/pull/101/commits/be7ca46b37eaffcbab8c64e6153ed8fae6ad9027

Reply:
It's not using the `errors.py` as expected, please review